makefile

```diff
@@ -38,7 +38,7 @@ install-aws: ## Install the aws cli
 
 install-prod: ## Install npm dependencies for the api, admin, and frontend apps in production environment
 	@echo "Installing Node dependencies"
-	@npm install
+	@npm install --production
```

Comment:
Are you sure about this ? We use npm install for deployments and we need dev deps to build the static elements